src/widget/systemtrayicon.cpp

```diff
@@ -233,13 +244,17 @@ void SystemTrayIcon::setContextMenu(QMenu* menu)
                 };
                 QImage* image = new QImage(a->icon().pixmap(64, 64).toImage());
                 if (image->format() != QImage::Format_RGBA8888_Premultiplied)
-                *image = image->convertToFormat(QImage::Format_RGBA8888_Premultiplied);
+                    *image = image->convertToFormat(QImage::Format_RGBA8888_Premultiplied);
                 GdkPixbuf* pixbuf = gdk_pixbuf_new_from_data(image->bits(), GDK_COLORSPACE_RGB, image->hasAlphaChannel(),
```

Comment:
**important** Will need to `memcpy` the `image->bits()` here, as these are shared implicitly and become invalid, when QImage is deleted.

After doing so, QImage can be created on stack again.